src/Microsoft.AspNet.Http.Abstractions/HttpContextAccessor.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+#if DNX451
+using System.Runtime.Remoting.Messaging;
+using System.Runtime.Remoting;
+#elif DNXCORE50
+using System.Threading;
+#endif
+
+namespace Microsoft.AspNet.Http.Internal
```

Comment:
Oh, I just realized this was in the Abstractions package. This implementation should move to the Http package.